### ISSUE TITLE
Platformer example clipping into the ground for 1 frame

### ISSUE THREAD

**Issue by gusinacio** [Trouv/bevy_ecs_ldtk#309]
If I jump from a a high altitude, when reaching the ground it clips through the ground 1 frame and then it gets back to "normal".

I'm using the Platformer example as a starting point and I don't know what part of rapier is causing this (or if is a problem of the example itself)

**Comment by Trouv** [comment]
Yes, the platformer example does have some jank like this. It is most likely our very basic *usage* of rapier that causes it, not rapier. It's not necessarily intended to be a perfect platformer, just something simple that touches many features of bevy_ecs_ldtk. I think users should be able to configure the player character more effectively using existing rapier components that minimize this sort of issue.

One thing I haven't tried using yet is rapier's [character controller](https://rapier.rs/docs/user_guides/bevy_plugin/character_controller). This might resolve some of the issues since it's an official rapier solution, and would probably help reduce much of the platformer example code. I might try rewriting the platformer example to use it someday.
